### PR TITLE
Make fiat conversions accurate

### PR DESCRIPTION
**Pull request by lukaw3d**

### CODE REVIEW

File: src/locales/__tests__/i18n-formatting.test.ts [hunk 1]
```diff
@@ -38,7 +38,7 @@ test('formatting fiat currency with i18n', () => {
     }) as string
   })
 
-  expect(result.current).toBe('$123.46')
+  expect(result.current).toBe('$111,222,333,444,555,666,777,888,999.11')
```
Comment (buberdds) [inline src/locales/__tests__/i18n-formatting.test.ts:41]: This one is failing

Comment (lukaw3d) [reply src/locales/__tests__/i18n-formatting.test.ts:41]: Ah. Need to upgrade everything to Node 19

Comment (lukaw3d) [reply src/locales/__tests__/i18n-formatting.test.ts:41]: (To match browser behaviour)

File: .github/workflows/ci-test.yml [hunk 2]
```diff
@@ -19,10 +19,10 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
-      - name: Set up Node.js 18
+      - name: Set up Node.js 19
```
Comment (buberdds) [inline .github/workflows/ci-test.yml:22]: Do we really want to use dev Node version that is already dead (no updates/no security updates)?

Comment (lukaw3d) [reply .github/workflows/ci-test.yml:22]: Oops, it's already dead

**Review by buberdds — APPROVED** [review]

**Pull request merged**